scripts/start.py

```diff
@@ -0,0 +1,204 @@
+        # Give webpack few seconds to do the initial compilation.
+        time.sleep(10)
+
+    python_utils.PRINT('Starting GAE development server')
+    background_processes.append(subprocess.Popen(
+        'python %s/dev_appserver.py %s %s --admin_host 0.0.0.0 --admin_port '
+        '8000 --host 0.0.0.0 --port %s --skip_sdk_update_check true %s' % (
+            common.GOOGLE_APP_ENGINE_HOME, clear_datastore_arg,
+            enable_console_arg,
+            python_utils.UNICODE(PORT_NUMBER_FOR_GAE_SERVER),
+            app_yaml_filepath), shell=True))
+
+    # Wait for the servers to come up.
+    while not common.is_port_open(PORT_NUMBER_FOR_GAE_SERVER):
+        time.sleep(1)
+
+    os_info = os.uname()
+    # Launch a browser window.
+    if os_info[0] == 'Linux' and not parsed_args.no_browser:
+        detect_virtualbox_pattern = re.compile('.*VBOX.*')
+        if list(filter(
+                detect_virtualbox_pattern.match,
+                os.listdir('/dev/disk/by-id/'))):
+            common.print_each_string_after_two_new_lines([
+                'INFORMATION',
+                'Setting up a local development server. You can access this '
+                'server',
+                'by navigating to localhost:%s in a browser window.'
+                % python_utils.UNICODE(PORT_NUMBER_FOR_GAE_SERVER)])
+        else:
+            common.print_each_string_after_two_new_lines([
+                'INFORMATION',
+                'Setting up a local development server at localhost:%s. '
+                % python_utils.UNICODE(PORT_NUMBER_FOR_GAE_SERVER),
+                'Opening a',
+                'default browser window pointing to this server'])
+            time.sleep(5)
+            background_processes.append(
+                subprocess.Popen([
+                    'xdg-open', 'http://localhost:%s/'
+                    % python_utils.UNICODE(PORT_NUMBER_FOR_GAE_SERVER)]))
+    elif os_info[0] == 'Darwin' and not parsed_args.no_browser:
+        common.print_each_string_after_two_new_lines([
+            'INFORMATION',
+            'Setting up a local development server at localhost:%s. '
+            % python_utils.UNICODE(PORT_NUMBER_FOR_GAE_SERVER),
+            'Opening a',
+            'default browser window pointing to this server.'])
+        time.sleep(5)
+        background_processes.append(
+            subprocess.Popen([
+                'open', 'http://localhost:%s/'
+                % python_utils.UNICODE(PORT_NUMBER_FOR_GAE_SERVER)]))
+    else:
+        common.print_each_string_after_two_new_lines([
+            'INFORMATION',
+            'Setting up a local development server. You can access this server',
+            'by navigating to localhost:%s in a browser window.'
+            % python_utils.UNICODE(PORT_NUMBER_FOR_GAE_SERVER)])
+
+    python_utils.PRINT('Done!')
+
+    for process in background_processes:
+        process.wait()
+
+
+if __name__ == '__main__':
+    main(args=None)
```

Comment:
Just leave this as `main()` here and in the other `if __name__ == '__main__'` blocks.

Reply:
Done